common/changes/@typespec/eslint-config-typespec/TypeSpecRename_2023-02-14-19-01.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/eslint-config-typespec",
+      "comment": "Renaming Product to TypeSpec",
```

Comment:
Minor but "Rename to TypeSpec" sounds better to me ("Product" is somewhat awkward in this context IMO).